src/locale/en.json

```diff
@@ -1,4 +1,6 @@
 {
+	"audience_add_name": "Add audience name",
+	"audience_edit_name": "Edit audience name",
```

Comment:
We should be more precise and consistent with our `en.json` keys. 

Maybe something along the lines of:
```
audience_add_name_title
audience_edit_name_title
```